apps/src/templates/school-association.module.scss

```diff
@@ -1,17 +1,43 @@
-@import "color.scss";
-@import "font.scss";
+@import 'color.scss';
+@import 'font.scss';
 
 .outerContainer {
-    text-align: center;
-    display: grid;
+  text-align: center;
+  display: grid;
+  padding: 10px;
+
+  h2 {
+    margin-bottom: 5px;
+  }
 }
 
 .inputContainer {
-    justify-self: center;
-    text-align: left;
-    width: 400px;
+  justify-self: center;
+  text-align: left;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+
+  & > * {
+    width: 100%;
+  }
+
+  & > input {
+    width: initial;
+    margin-top: 5px !important;
```

Comment:
These important tags are necessary to override these specific stylings from the component library stylesheets

Reply:
I don't think .25rem actually matches... can I do .37rem to match the dropdown paddings?

Reply:
Oh yeah, that's fine!

Reply:
Updated it and changed the pic in the pr description!

Reply:
Woo hoo!! 🎉